### PR TITLE
Add ANN impl

### LINKED ISSUE #12: Implement a Second Algorithm for Comparison
Implement a second algorithm to compare to Bayes' net. You're free to try to implement either ANN (#3) or k-means(#4) (or both, make another task for the other algorithm).

Try to use the dataset cleaned up in #8 , maybe the dataset needs to be a different format for the second algorithm - you'll need to make a copy of the dataset with the format needed and use it.

Use the `sci-kit` python module and figure out how to make your necessary model and use it to predict disease from a set of symptoms. (Or some other library... maybe you'll use another GitHub repository's code - be sure to give credit and that their license allows us to use their code for academic purposes)

This task is done when:
- [ ] You've figured out how to use one of the two algorithms to predict diseases from a set of symptoms
- [ ] You've implemented or pulled together existing code to implement your chosen algorithm
- [ ] You've unit tested your implementation against the test data noted in #8. (Or otherwise, maybe the code you're using has unit tests already).

### PR DESCRIPTION
**Pull request by alwinbaby**
Closes #12

### CODE REVIEW

File: csv/Example_ANN_Training_Samples.csv [hunk 1]
```diff
@@ -0,0 +1,830 @@
+0,1,0,0,0,Acne
```
Comment (holtzmak) [inline csv/Example_ANN_Training_Samples.csv:1]: @alwinbaby could you confirm this was the dataset you used for training?

File: src/ann.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        sweep_y = y
+        x_train, x_test, y_train, y_test = train_test_split(
+            sweep_x, sweep_y, test_size=0.20
+        )
+        # Feature Scaling for uniform evaluation
+        scaler = StandardScaler()
+        scaler.fit(x_train)
+        x_train = scaler.transform(x_train)
+        x_test = scaler.transform(x_test)
+        return x_train, x_test, y_train, y_test
```
Comment (holtzmak) [inline src/ann.py:38]: I wasn't sure how you intended to use this, so I adjusted the returns the way I thought you intended to use them. @alwinbaby does this look like what you wanted?

**Review by holtzmak — APPROVED** [review]
If my adjustments look right to you @alwinbaby , you can merge this right away and be done with it.

File: csv/Example_ANN_Training_Samples.csv (hunk 1 above)
Comment (alwinbaby) [reply csv/Example_ANN_Training_Samples.csv:1]: Yup this is the one i used

File: src/ann.py (hunk 2 above)
Comment (alwinbaby) [reply src/ann.py:38]: As long as other def functions can use those its good

**Comment by alwinbaby** [comment]
Merging ANN Impl

**Pull request merged**